spec/formatting.md

```diff
@@ -372,6 +372,30 @@ to provide the _pattern_ for the formatting operation.
 This is done by ordering and filtering the available _variant_ statements
 according to their _key_ values and selecting the first one.
 
+> [!NOTE]
+> At least one _variant_ is required to have all of its _keys_ consist of
+> the fallback value `*`.
+> Some _selectors_ might be implemented in a way that the key value `*`
+> cannot be selected in a _valid_ _message_.
+> In other cases, this key value might be unreachable only in certain locales.
+> This could result in the need in some locales to create
+> one or more _variants_ that do not make sense grammatically for that language.
+> > For example, in the `pl` (Polish) locale, this _message_ cannot reach
+> > the `*` _variant_:
+> > ```
+> > .match {$num :integer}
+> > 0    {{ }}
+> > one  {{ }}
+> > few  {{ }}
+> > many {{ }}
+> > *    {{Only used by fractions in Polish.}}
+> > ```
```

Comment:
Would this be a simpler example for what's being demostrated here?

```suggestion
> > For example, in the `en` (English) locale, this _message_ cannot reach
> > the `*` _variant_ unless `$num` is not a valid number:
> > ```
> > .match {$num :number}
> > one   {{ }}
> > other {{ }}
> > *     {{ }}
> > ```
```

Reply:
Do we really want to promote the use of `other` with the number functions? In English? I thought the Polish example was clever because it naturally exposes the issue at hand (and begs the question "what does that message say?"). But happy to accept this too...

Reply:
As I've understood, in Polish `many` _is_ the fallback category, so explicitly including it (as in your example) is not really any different from my proposal.